docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -35,14 +34,14 @@ To deploy all the prerequisities for building Open Enclave, you can run the foll
 
 ```powershell
 cd scripts
-.\install-windows-prereqs.ps1 -InstallPath C:\path\to\where\you\would\like\to\install\intel_and_dcap_nuget_packages -LaunchConfiguration SGX1 -DCAPClientType None
+.\install-windows-prereqs.ps1 -InstallPath C:\path\to\where\you\would\like\to\install\intel_nuget_packages -LaunchConfiguration SGX1 -DCAPClientType None
```

Comment:
You could change this back to intel_and_dcap_nuget_packages, but it really doesn't matter! It's just a fake path that a user should replace, so it's irrelevant